rtb/verbali/interni/2024-02-24/main.tex

```diff
@@ -0,0 +1,146 @@
+    \includegraphics[width=\linewidth]{logo rotondo.jpg}
+\end{minipage}\hfil
+\begin{minipage}{0.55\linewidth}
+\textbf{Gruppo:} SWEet16 \\
+\textbf{Email:} 
+\href{mailto:sweet16.unipd@gmail.com}{\nolinkurl{sweet16.unipd@gmail.com}}
+\end{minipage}
+
+\vspace{15mm}
+
+\begin{center}
+\begin{Huge}
+        \textbf{Verbale Interno} \\
+        \vspace{4mm}
+        \textbf{24 Febbraio 2024}
+\end{Huge}
+
+\vspace{20mm}
+
+\begin{large}
+\begin{spacing}{1.4}
+\begin{tabular}{c c c}
+   Redattori:  &  Alberto M. & \\
+   Verificatori: & Alex S. & \\
+   Amministratore: &  Alex S. & \\
+   Destinatari: & T. Vardanega & R. Cardin \\  
+   Versione: & 1.0.0 & 
+\end{tabular}
+\end{spacing}
+\end{large}
+\end{center}
+
+\pagebreak
+
+
+\begin{huge}
+    \textbf{Registro delle modifiche}
+\end{huge}
+\vspace{5pt}
+
+\begin{tblr}{
+colspec={|X[1.5cm]|X[2cm]|X[2.5cm]|X[2.5cm]|X[5cm]|},
+row{odd}={bg=white},
+row{even}={bg=lightgray},
+row{1}={bg=black,fg=white}
+}
+    Versione & Data & Autore & Verificatore & Descrizione \\
+    \hline
+    1.0.0 & 2023/12/16 & Alex S. & & Approvazione per rilascio \\
+    \hline
+     0.1.0 & 2023/12/13 & Alberto M. & Alex S. & Prima stesura del documento \\
+     \hline
+\end{tblr}
+
+\pagebreak
+
+\section{Partecipanti}
+Di seguito i nomi dei partecipanti con le rispettive matricole: \\
+\vspace{5mm}
+
+\begin{table}[h]
+\begin{tblr}{
+colspec={X[5cm]X[5cm]},
+row{odd}={bg=moregreen},
+row{even}={bg=lightgreen},
+row{1}={bg=darkgreen,fg=white}
+}
+    Nome & Matricola \\
+    Alberto Cinel & 1142833 \\
+    Bilal El Moutaren & 2053470 \\
+    Alberto Michelazzo & 2010007 \\
+    Alex Scantamburlo & 2042326 \\
+    Iulius Signorelli & 2012434 \\
+    Giovanni Zuliani & 595900 
+\end{tblr}
+\end{table}
+
+\vspace{10pt}
+
+\textbf{Inizio incontro:} Ore 14:30 \newline
+\textbf{Fine incontro:} Ore 15:30  \newline
+
+\pagebreak
+
+\section{Sintesi ed elaborazione incontro}
+
+\subsection{Incontro con proponente}
+L'incontro si è avviato con la condivisione dei feedback ricevuti dal proponente alla visione del PoC$^{G}$, durante l'incontro del 23 febbraio, 
+visto che non tutti i componenti del gruppo erano presenti perché in concomitanza con il diario di bordo settimanale.
+
+\subsection{Riunioni settimanali}
+Essendo la sessione invernale degli esami conclusa è stato deciso di ricominciare ad effettuare delle riunioni interne a cadenza settimanale, il giovedì sera alle 21. \\
+Precedentemente alla sessione degli esami gli incontri erano programmati per il martedì sera, ma spostandoli verso la fine della settimana abbiamo la possibilità di fare un resoconto delle attività svolte più completo,
+oltre che a semplificare la stesura del diario di bordo, essendo quest'ultimo ogni venerdì pomeriggio.
+
+\subsection{Modalità stesura documentazione}
+Tutti i componenti del gruppo sono stati allineati riguardo alla nuova procedura di stesura della documentazione, tramite la suddivisione dei documenti in sezioni, 
```

Comment:
due punti al posto della virgola:"
nuova procedura ... documentazione: ....
"

o se no un elenco puntato lo vedo meglio:
- suddivisione in sezioni
- apertura di una issue per sezione
- merge tramite pull req per sezione